src/wazuh_modules/wm_ciscat.c

```diff
@@ -1625,6 +1625,14 @@ void wm_ciscat_destroy(wm_ciscat *ciscat) {
         free(cur_eval);
     }
 
+    if (ciscat->ciscat_path){
```

Comment:
Please, change it, and use os_free that already does that check and we avoid repeating code.